src/components/tag/tag/readme.md

```diff
@@ -32,6 +32,16 @@ The component can be displayed in `checked` or `disabled` state using the self-n
 <sbb-tag disabled value="All" icon-name="circle-information-small">All</sbb-tag>
 ```
 
+## Style
+
+The component has two sizes, named `m` (default) and `s`.
```

Comment:
```suggestion
The component has two sizes, named `m` (default) and `s`. The `size` property can also be set on the `sbb-tag-group` where it will be applied to all tags inside the group.
```